.github/workflows/gitlab-ci.yaml

```
@@ -78,4 +78,12 @@ jobs:
 
       - name: Run tests
         run: |
-          python manage.py test
\ No newline at end of file
+          python manage.py test
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)